File: 1426-FindNUniqueIntegersSumUpToZero/1426-FindNUniqueIntegersSumUpToZero.java
// Last updated: 18/10/2025, 09:03:03
class Solution {
    public int[] sumZero(int n) {
        int arr[] = new int[n];
       int num = 1;
       for(int i = 0; i  < n / 2;i++){
        arr[i] = num;
        arr[n - i - 1] = -num;
        num++;
       }

       return arr;
    }
}

File: 1426-FindNUniqueIntegersSumUpToZero/1426-FindNUniqueIntegersSumUpToZero.cpp
// Last updated: 18/10/2025, 09:02:59
class Solution {
public:
    vector<int> sumZero(int n) {
        vector<int>result(n,0);
        int i = 0;
        int j = n - 1;
        int start = 1;
        while(i < j){
            result[i] = start;
            result[j] = -start;
            start++;

            i++;
            j--;
        }
        return result;
    }
};